Plutarch/Field.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Plutarch.Field (PField (..), FieldName (..), pfield) where
```

Comment:
?

Reply:
Did your merge go bad?

Reply:
looks like it, I have no idea how this happened.

Reply:
I think vscode must've restored an old cached file without asking :upside_down_face: